ap_src/ap_app/utils/user_profile.py

```diff
@@ -0,0 +1,15 @@
+# Models
+from ..models import (UserProfile)
```

Comment:
You shouldn't need the () around UserProfile

Reply:
This was code copy and pasted from elsewhere in the codebase. The brackets will be removed.